api/consensus.go

```diff
@@ -19,6 +21,55 @@ type ConsensusGET struct {
 	Difficulty   types.Currency    `json:"difficulty"`
 }
 
+type ConsensusFileContract struct {
+	FileSize           uint64                         `json:"filesize"`
+	FileMerkleRoot     crypto.Hash                    `json:"filemerkleroot"`
+	WindowStart        types.BlockHeight              `json:"windowstart"`
+	WindowEnd          types.BlockHeight              `json:"windowend"`
+	Payout             types.Currency                 `json:"payout"`
+	ValidProofOutputs  map[string]types.SiacoinOutput `json:"validproofoutputs"`
+	MissedProofOutputs map[string]types.SiacoinOutput `json:"missedproofoutputs"`
```

Comment:
I don't think using a map here is the right move, maps in go are randomly ordered, but valid and missed proofs have a specific order, and the order is meaningful.